build/parser.mjs

```diff
@@ -869,7 +869,7 @@ class Parser {
   addWeaponWikiaData(item, wikiaItem) {
     item.attacks = wikiaItem.attacks;
     item.ammo = wikiaItem.ammo;
-    item.marketCost = wikiaItem.marketCost;
+    item.marketCost = wikiaItem.marketCost || undefined;
```

Comment:
I just remembered if marketCost is 0 it'll be assigned to undefined
```suggestion
    item.marketCost = typeof wikiaItem.marketCost === 'number' ? wikiaItem.marketCost : undefined;
```

Reply:
If its 0 it already is undefined for all purposes. If its 0 plat, there's no point knowing its 0...

Reply:
some items may be free, i don't remember exactly if it already happened but it may be a thing

Reply:
Items that are free are always set to 1 credit. I wanted to say that this morning but I was waking up and figured being information complete is just as important.

But to be honest the information for market cost comes from the wikia so you also have to consider the one person who sets the market cost to 0.